doc/source/GettingStarted/SettingUpGafferCommand/index.md

```diff
@@ -0,0 +1,142 @@
+# Setting Up the "gaffer" Command #
```

Comment:
It's a compromise: ideally, I'd use the inline code tag, but that messes with the titles in the TOC, hence the quotation marks. 

I want to reduce confusion between Gaffer the app, API and node classes that start with the word Gaffer, and commands. It helps with skimming. Since the command is lowercase (as an env var it could be anything, but the script itself is `bin/gaffer`, so I'm sticking to the most 1:1 approach), and case matters in Linux, it's more accurate this way.

Reply:
Oh, sorry, I didn't mean capitalization of "gaffer", I meant of Up and Command

Reply:
I'm using title case, mostly out of habit. You'll see a lot of web content use sentence case, but title case is still fairly popular in more formal writing. I'm not married to either form, and if you prefer I could make a PR undoing all of them (after we merge my PRs – all my H1s and H2s are like this, and rebasing everything at this stage will be a pain). Of possible note: I've been using it everywhere so far, so I wonder if you noticed it this time because it was juxtaposed with "gaffer"?